containers/daap-delete-table-for-data-product/src/var/task/delete_table.py

```diff
@@ -0,0 +1,123 @@
+import os
+from http import HTTPStatus
+
+import boto3
+from botocore.exceptions import ClientError
+from data_platform_api_responses import format_error_response
+from data_platform_logging import DataPlatformLogger
+from data_platform_paths import DataProductElement
+from data_product_metadata import DataProductMetadata, DataProductSchema
+
+logger = DataPlatformLogger(
+    extra={
+        "image_version": os.getenv("VERSION", "unknown"),
+        "base_image_version": os.getenv("BASE_VERSION", "unknown"),
+    }
+)
+
+s3_client = boto3.client("s3")
+glue_client = boto3.client("glue")
+athena_client = boto3.client("athena")
+
+
+def s3_recursive_delete(bucket, prefix) -> None:
```

Comment:
Just wondering if you tried using the `filter` method described in this stack overflow answer? https://stackoverflow.com/questions/11426560/amazon-s3-boto-how-to-delete-folder

Seems like you can do the same thing more simply, but I haven't tried it out